index.js

```diff
@@ -0,0 +1,130 @@
+
+const URL = "https://trektravel.herokuapp.com/trips"
+
+
+const loadTrips = () => {
+
+  tripList = $('#trip-list');
+  tripList.empty();
+
+
+  axios.get(URL)
+    .then((response) => {
+      response.data.forEach((trip) => {
+        tripList.append(`<li id="${trip.id}"><a href="#">${trip.name}</a></li>`);
+        $("#" + trip.id).click( () => {
+          loadDetails(trip.id);
+          createForm(trip.id, trip.name) })
+
+
+        })
+        tripList.prepend("<h2>All Trips</h2>")
+        $(".current-trips").css("border-style", "solid");
+    })
+    .catch((error) => {
+      console.log(error);
+    });
+
+};
+
+
+const loadDetails = (id) => {
+  const tripDetails = $('#detail-list');
+  tripDetails.empty()
+
+  axios.get(URL+"/"+ id)
+  .then((response) => {
+      let titles = ["name", "continent", "category", "weeks", "cost", "about"]
+      titles.forEach((title) => {
+        tripDetails.append(`<li><b>${title}:</b> ${response["data"][title]}</li>`);
+      })
+      tripDetails.prepend(`<h2>Trip Details</h2>`)
+      $(".details").css("border-style", "solid");
+
+  })
+  .catch((error) => {
+    console.log(error);
+  });
+  
+};
+
+
+const createForm = (id, name) => {
+  const form = $('#form');
+  form.empty()
+  const buildFormHtml = `<h2>Reserve a Spot on ${name} </h2>
+  <div >Name: <input type="text" name="name"></div>
+  <div>Email: <input type="text" name="email"></div>
+  <input type="submit" name="reserve" value="Reserve" />`
+
+  form.html(buildFormHtml)
+  $(".reservation-form").css("border-style", "solid");
+
+  $('#form').submit( (event) => {
+    event.preventDefault();
+    createReservation(id, name)
```

Comment:
When reserving a second trip, your site makes two POST requests. To reproduce:
1. Open page
1. Open chrome dev tools to the "Network" tab
1. Click See All Trips
1. Click on the first trip (Cairo to Zanzibar)
1. Make a reservation and submit
1. Click on the second trip (Everest Base Camp Trek)
1. Make a reservation and submit
1. Observe that on submit, 2 POST requests were made to trek API's /reservations endpoint

The reason for this is that you reuse the same reservation form for each trip, but re-attach the `submit` handler every time you show details for a trip.